### PR TITLE
Order core and controller classes

### PR DESCRIPTION
**Pull request by SaulCM**
Moved classes between core and controllers and solved the issue with the converter threads not finishing before executing the next code.

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.fundacion.jala.converter.core;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java]: Copyright must be on line 1

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * *  This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ */
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:12]: Add a description of the class

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+class ThreadHandler extends Thread {
+
+    InputStream input;
+
+    ThreadHandler(InputStream input, String name) {
+        super(name);
+        this.input = input;
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:25]: Add javadoc

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 4]
```diff
@@ -29,6 +29,10 @@
     private static final int WAIT_TIME = 7000;
     private static final int INIT_NUMBER = 20;
     private static final Logger LOGGER = LogManager.getLogger();
+    private Result result;
+    private String thumbnailFilename;
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: delete blank line

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 5]
```diff
@@ -90,19 +93,22 @@ private String changeResolution() {
...
         String name = getOutputFileName().substring(0, getOutputFileName().lastIndexOf("."));
         String startCommand = "ffmpeg -i ";
         String outputCommand = pathOutput + output + "\"" + " -ss 00:00:01 -vframes 1 -s 128x128 " +
-                               pathOutput + name + ".png\"";
+                               pathOutput + name + PNG_FORMAT;
         String thumbnailCommand = startCommand + outputCommand;
+
+        Process process = Runtime.getRuntime().exec("cmd /c " + thumbnailCommand);
+        ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
+        errorHandler.start();
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:102]: Good job

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 6]
```diff
@@ -130,7 +136,7 @@ private String changeFrameRate() {
         int frameRate = parameter.getFrameRate();
         String frameCommand;
         if (frameRate > INIT_NUMBER) {
-            frameCommand = " -r " + frameRate + " -y ";
+            frameCommand = " -filter:v fps=" + frameRate + " ";
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:136]: :V

File: src/main/java/org/fundacion/jala/converter/controller/security/MyUserDetailsService.java [hunk 7]
```diff
@@ -8,14 +8,15 @@
...
-package org.fundacion.jala.converter.service;
+package org.fundacion.jala.converter.controller.security;
 
 import org.fundacion.jala.converter.models.AuthenticationRequest;
 import org.springframework.security.core.userdetails.User;
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 import org.springframework.stereotype.Service;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/controller/security/MyUserDetailsService.java]: the line 19 should be removed

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+public class Result {
+    private String filename;
+    private String path;
+    private String processId;
+    private String textContent;
+
+    public String getFilename() {
+        return filename;
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java:30]: JavaDoc is missing in the class

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java [hunk 9]
```diff
@@ -83,4 +88,12 @@ public String makePythonFile(final String code) {
             return exception.getMessage();
         }
     }
+
+    /**
+     * Returns the result object for the operation.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java:91]: is missing a line empty between 93 and 94

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+ * *  This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ */
+
+class ThreadHandler extends Thread {
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:14]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 11]
```diff
@@ -37,33 +41,32 @@ public VideoConverter(final VideoParameter videoParameter) {
...
         format = parameter.getOutputFormat();
         output = adaptPath.substring((adaptPath.lastIndexOf("\\") + 1), adaptPath.lastIndexOf(".") + 1) + format;
         setOutputFileName(output);
         pathOutput = adaptPath.substring(0, (adaptPath.lastIndexOf("archive"))) + "archive\\";
         String ffmpegCommand = startFirstCommand + adaptPath + " ";
         String parameters = changeResolution() + changeFrameRate() + removeAudio();
-        String theCommand = ffmpegCommand + parameters + pathOutput + output + "\"" + " -y";
-        System.out.println(theCommand);
+        String theCommand = ffmpegCommand + parameters + pathOutput + output  + "\"";
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 12]
```diff
@@ -90,19 +93,22 @@ private String changeResolution() {
...
      */
-    private void generateAThumbnail() {
+    private void generateAThumbnail() throws IOException{
         String name = getOutputFileName().substring(0, getOutputFileName().lastIndexOf("."));
         String startCommand = "ffmpeg -i ";
         String outputCommand = pathOutput + output + "\"" + " -ss 00:00:01 -vframes 1 -s 128x128 " +
-                               pathOutput + name + ".png\"";
+                               pathOutput + name + PNG_FORMAT;
         String thumbnailCommand = startCommand + outputCommand;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java [hunk 13]
```diff
@@ -153,4 +159,12 @@ public void setOutputFileName(final String newOutputFileName) {
     public String getOutputFileName() {
         return outputFileName;
     }
+
+    /**
+     * Returns the object result for the operation.
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:161]: there should be a space between the lines 164 and 165

File: src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java [hunk 14]
```diff
@@ -8,11 +8,13 @@
...
  */
-package org.fundacion.jala.converter.service.javacompiler;
+package org.fundacion.jala.converter.core.javacompiler;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.fundacion.jala.converter.models.parameter.JavaParameter;
+import org.fundacion.jala.converter.core.parameter.JavaParameter;
+import org.fundacion.jala.converter.core.results.Result;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java]: this line 17 should be removed

File: src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java [hunk 15]
```diff
@@ -62,4 +68,12 @@ public String javaCompiler(final JavaParameter newJavaParameter) {
             }
         }
     }
+
+    /**
+     * Returns the result object for the operation.
+     * @return compilerResult
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java:74]: here there should be a space between the lines 73 and 74

File: src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java [hunk 16]
```diff
@@ -8,14 +8,14 @@
  *
  * @author Juan Pablo Gonzales Alvarado
  */
-package org.fundacion.jala.converter.models.parameter;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java]: this line 11 should be removed

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java (hunk 1 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java (hunk 2 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:12]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java (hunk 3 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:25]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 4 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: Done.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * *  This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Saul Caspa Miranda
+ */
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java:12]: the package should be after the copyright

Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java:12]: And the copyright should start from line 1

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+public class Result {
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java:16]: is missing javadoc of the class

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 19]
```diff
@@ -27,7 +27,7 @@
 import java.io.IOException;
 import java.security.NoSuchAlgorithmException;
 
-import static org.fundacion.jala.converter.service.ChecksumService.getFileChecksum;
+import static org.fundacion.jala.converter.core.ChecksumService.getFileChecksum;
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:28]: in that file the lines 18 and 29 should be removed

File: src/main/java/org/fundacion/jala/converter/controller/security/MyUserDetailsService.java (hunk 7 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/controller/security/MyUserDetailsService.java]: Ok, done.

File: src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java [hunk 20]
```diff
@@ -39,8 +39,8 @@ private ChecksumFacade() {
      * @return object of ParameterOutputChecksum.
      * @throws IOException is a exception when invalid file's path.
      */
-    public static ParameterOutputChecksum getChecksum(final String checksum,
-                                                      final MultipartFile file) throws IOException {
+    public static ParameterOutputChecksum getChecksum(final String checksum, final MultipartFile file)
+            throws  IOException {
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java]: I think there is an extra space, maybe you could delete it.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 8 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java:30]: Done.

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java (hunk 9 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java:91]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java (hunk 10 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java:14]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 11 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: Done.

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 12 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java]: Line removed.

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 13 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:161]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java (hunk 14 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java (hunk 15 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java:74]: Line removed.

File: src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java (hunk 16 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java]: Line removed.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 17 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java:12]: Done.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 18 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java:16]: Solved.

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 19 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:28]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java (hunk 20 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/facade/ChecksumFacade.java]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 13 above)
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:161]: Add a new line before return.

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java (hunk 9 above)
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java:91]: Add a line before the return, and also return and parameters has a dot at the end.

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java [hunk 21]
```diff
@@ -112,18 +118,26 @@ private void safeInfo(final String name, final String text) {
...
             bufferedWriter.close();
+            result.setFilename(name + TXT_EXTENSION);
         } catch (Exception e) {
             LOGGER.error("Execute Exception to Safe text in a file");
             e.printStackTrace();
         }
         LOGGER.info("finish");
     }
+    /**
+     * Returns the object result for the operation.
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/ExtractText.java:135]: Add a line before return.

File: src/main/java/org/fundacion/jala/converter/core/ExtractMetadata.java [hunk 22]
```diff
@@ -87,4 +93,12 @@ public static void extractMetadata(final String metadata, final String outputFil
             extractMetadata.extractMetadata();
         }
     }
+
+    /**
+     * Returns the object result for the operation.
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/ExtractMetadata.java:98]: Add a line before return.

File: src/main/java/org/fundacion/jala/converter/core/AudioConverter.java [hunk 23]
```diff
@@ -63,4 +66,12 @@ public String getOutputFileName() {
         return this.outputFileName;
     }
 
+    /**
+     * Returns the object result for the operation.
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/core/AudioConverter.java:72]: Add a line before the return

File: src/main/java/org/fundacion/jala/converter/core/VideoConverter.java (hunk 13 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/VideoConverter.java:161]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java (hunk 21 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ExtractText.java:135]: Line added.

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java (hunk 9 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java:91]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ExtractMetadata.java (hunk 22 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ExtractMetadata.java:98]: Line added.

File: src/main/java/org/fundacion/jala/converter/core/AudioConverter.java (hunk 23 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/AudioConverter.java:72]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 24]
```diff
@@ -0,0 +1,84 @@
...
+     */
+    public String getTextContent() {
+        return textContent;
+    }
+
+    /**
+     * Sets the text content for the result.
+     * @param textContent
+     */
+    public void setTextContent(String textContent) {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java]: Add final before String

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 25]
```diff
@@ -0,0 +1,84 @@
...
+ * This class stores the return variables for a converter and compiler result.
+ */
+public class Result {
+    private String filename;
+    private String path;
+    private String processId;
+    private String textContent;
+
+    /**
+     * Returns the filename.
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java:22]: Add line between 22 and 23

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 26]
```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java:2]: Add blank line between 2 and 3

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 24 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java]: Solved

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 26 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java:2]: Solved.

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java [hunk 27]
```diff
@@ -0,0 +1,40 @@
...
+        try {
+            int c;
+            while ((c = input.read()) != -1) {
+                System.out.write(c);
+            }
+        } catch (Throwable t) {
+            t.printStackTrace();
+        }
+    }
+
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java]: Delete blank line

**Review by Paola-Xaq — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java (hunk 27 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ThreadHandler.java]: Done.

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java [hunk 28]
```diff
@@ -8,13 +8,15 @@
...
+package org.fundacion.jala.converter.core;
 
 import net.sourceforge.tess4j.Tesseract;
 import net.sourceforge.tess4j.TesseractException;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.fundacion.jala.converter.models.parameter.ExtractTextParameter;
+import org.fundacion.jala.converter.core.parameter.ExtractTextParameter;
+import org.fundacion.jala.converter.core.results.Result;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/ExtractText.java]: This line should be removed

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java [hunk 29]
```diff
@@ -8,11 +8,13 @@
...
  */
-package org.fundacion.jala.converter.service;
+package org.fundacion.jala.converter.core;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.fundacion.jala.converter.models.parameter.PythonParameter;
+import org.fundacion.jala.converter.core.parameter.PythonParameter;
+import org.fundacion.jala.converter.core.results.Result;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/RunCommand.java [hunk 30]
```diff
@@ -8,7 +8,8 @@
  *
  * @author Edson Añawaya Rios
  */
-package org.fundacion.jala.converter.service;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/RunCommand.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/AudioConverter.java [hunk 31]
```diff
@@ -63,4 +68,13 @@ public String getOutputFileName() {
...
+     *
+     * @return converterResult.
+     */
+    public Result getResult() {
+        return result;
+    }
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/AudioConverter.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java [hunk 32]
```diff
@@ -28,6 +30,8 @@
     private String language;
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/ExtractText.java:28]: In this file the line 29 should be removed

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java [hunk 33]
```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * *  This software is the confidential and proprietary information of Fundacion Jala
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/results/Result.java]: In this line the "* " should be removed

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java [hunk 34]
```diff
@@ -8,14 +8,11 @@
  *
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java:8]: In this file in line 4 should be removed the "* " before This

File: src/main/java/org/fundacion/jala/converter/core/parameter/VideoParameter.java [hunk 35]
```diff
@@ -7,14 +7,11 @@
  *
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/parameter/VideoParameter.java:8]: in this file should have a empty line between line 2 and 3

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java (hunk 28 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ExtractText.java]: Done

File: src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java (hunk 29 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/PythonCompiler.java]: Line removed.

File: src/main/java/org/fundacion/jala/converter/core/RunCommand.java (hunk 30 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/RunCommand.java]: Removed.

File: src/main/java/org/fundacion/jala/converter/core/AudioConverter.java (hunk 31 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/AudioConverter.java]: Removed.

File: src/main/java/org/fundacion/jala/converter/core/ExtractText.java (hunk 32 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/ExtractText.java:28]: Removed.

File: src/main/java/org/fundacion/jala/converter/core/results/Result.java (hunk 33 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/results/Result.java]: Removed.

File: src/main/java/org/fundacion/jala/converter/core/parameter/VideoParameter.java (hunk 35 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/parameter/VideoParameter.java:8]: Removed.

File: src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java (hunk 34 above)
Comment (SaulCM) [reply src/main/java/org/fundacion/jala/converter/core/parameter/Parameter.java:8]: Done.

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**